tasks/install.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: download latest dokuwiki
+  get_url:
+    url: https://download.dokuwiki.org/src/dokuwiki/dokuwiki-stable.tgz
+    dest: /tmp/dokuwiki-stable.tgz
+ 
+- name: unarchive dokuwiki-stable.tgz to {{ dokuwiki_install_path }}
+  unarchive:
+    src: /tmp/dokuwiki-stable.tgz
+    dest: "{{ dokuwiki_install_path | dirname }}"
+    owner: www-data
+    remote_src: yes
+    list_files: yes
+  register: file_list
+
+- name: remove stupid version name from folder path
+  file:
+    src: "{{ dokuwiki_install_path | dirname }}/{{ file_list['files'][0] }}"
+    dest: "{{ dokuwiki_install_path }}"
+    owner: www-data
+    state: link
+
+- name: Install xml-php package used by dokuwiki.
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - php-ldap
+    - php-xml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Copy Logo to Server
+  copy:
+    src: files/logo.png
+    dest: "{{dokuwiki_install_path}}/data/media/wiki/logo.png"
+    owner: www-data
+
+- name: Remove default plugins
+  file:
+    path: '{{dokuwiki_install_path}}/lib/plugins/{{ item }}'
+    state: absent
+  with_items:
+    - authpdo
+    - authmysql
+    - authpgsql
+    - authplain
+    - authad
+
+- name: set acls
+  copy:
+    src: 'files/acl.auth.php'
+    dest: '{{dokuwiki_install_path}}/conf/acl.auth.php'
+
+- name: copy data 
+  synchronize:
+    src: '{{ dokuwiki_data_dir }}'
+    dest: '{{dokuwiki_install_path}}/data'
+  when: dokuwiki_data_dir is defined
+
+- name: Fix ldap connections for »ldaps://« server uri
```

Comment:
You could mention such hacks in the readme, with reference to the bug report or version.

Reply:
can we convert this to an issue that pi needs to fix since I did non of that?

Reply:
Do you know how many ignored issues we already have?
It would be really nice to know what the problem is.

Also, this task should only be executed if LDAP authentication is used.

Reply:
yeah but i cant fix this so either we leave this pull request open for ever or we open up an issue